server/config/auth/jwt.js

```diff
@@ -1,10 +1,10 @@
 import passport from 'passport';
 import jwt from 'jsonwebtoken';
-import dotenv from 'dotenv';
+// import dotenv from 'dotenv';
```

Comment:
Just commenting these out for now to highlight where they exist currently and to show how they would move to the middleware file.

Reply:
Remove these since it's working in middleware.

Reply:
Done